C2CRIBuildDir/projects/C2C-RI/src/RICenterServices/src/org/enterprisepower/net/portforward/Listener.java

```diff
@@ -32,9 +32,8 @@
  * 
  */
 public class Listener implements Runnable {
-//	private static Log log = LogFactory.getLog(Listener.class);
+    private static Log log = LogFactory.getLog(Listener.class);
```

Comment:
Should this be from ` private static final Logger log = LogManager.getLogger(Listener.class);`?

Reply:
Yes it should use the LogManager interface, good catch